### PR TITLE
Add Thor hero cards and fix a typo in another one of the pack

### PR DESCRIPTION
**Pull request by simongeorges**

### CODE REVIEW

**Comment by zzorba** [comment]
Thanks for typing these up.

How certain are we about the Thor cards (both positions and quantities)?  Based on how the DB is built, once we put this out it will be pretty challenging to make a correction to any decks built with the wrong numbers.

File: pack/thor.json [hunk 1]
```diff
@@ -1,4 +1,203 @@
...
+		"faction_code": "hero",
+		"flavor": "\"For honor! For glory! For Asgard!\" -- Thor",
+		"name": "For Asgard!",
+		"pack_code": "thor",
+		"position": 4,
+		"quantity": 1,
+		"resource_physical": 1,
+		"set_code": "thor",
+		"set_position": 5,
+		"text": "<b>Alter-Ego Action</b>: Search your deck and discard pile for a card with the <b>ASGARD</b> trait and add it to your hand. Shuffle your deck.",
```
Comment (zzorba) [inline pack/thor.json]: For traits in card text we do the following
`[[Asgard]]`

And the renderer will take care of making it bold/italic

File: pack/thor.json [hunk 2]
```diff
@@ -1,4 +1,203 @@
...
+		"text": "Restricted. <i>(Max 2 restricted cards per player.)</i>\nThor gets +1 ATK and gains the <b>AERIAL</b> trait.",
+		"traits": "Asgard. Weapon.",
+		"type_code": "upgrade"
+	},
+	{
+		"code": "06010",
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"flavor": "\"Brothers and sisters -- prepare yourselves. Today we go to WAR!\" -- Thor",
```
Comment (zzorba) [inline pack/thor.json]: Needs an `"is_unique": true,` on the helmet

File: pack/thor.json [hunk 3]
```diff
@@ -1,4 +1,203 @@
...
+		"faction_code": "hero",
+		"flavor": "\"I say thee, nay!\" -- Thor",
+		"name": "Lightning Strike",
+		"pack_code": "thor",
+		"position": 6,
+		"quantity": 2,
+		"resource_mental": 1,
+		"set_code": "thor",
+		"set_position": 9,
+		"text": "<b>Hero Action</b>: Spend X [energy] ressources → deal X damages to the villain and each minion engaged with you. This attack ignores tough statuses card if you have the <b>AERIAL</b> trait.",
```
Comment (zzorba) [inline pack/thor.json]: `[[Aerial]]`

File: pack/thor.json [hunk 4]
```diff
@@ -1,4 +1,203 @@
...
+		"flavor": "Whosoever holds this hammer, if he be worthy, shall possess the power of Thor.",
+		"is_unique": true,
+		"name": "Mjolnir",
+		"pack_code": "thor",
+		"position": 9,
+		"quantity": 1,
+		"resource_energy": 1,
+		"set_code": "thor",
+		"set_position": 14,
+		"text": "Restricted. <i>(Max 2 restricted cards per player.)</i>\nThor gets +1 ATK and gains the <b>AERIAL</b> trait.",
```
Comment (zzorba) [inline pack/thor.json]: `[[Aerial]]`

**Comment by simongeorges** [comment]
Yeah, I'm inclined to trust them, but no guarantee… Someone worked out the Miss Marvel pack as well, but without any guarantee, so seeing your remark here about deck updates, I'm wondering if I should type them up…

Anyway, I fixed the file following your remarks.

**Pull request closed without merging**